recipes/lerclib/all/patches/add-CMakeLists-and-static.patch

```diff
@@ -0,0 +1,61 @@
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,24 @@
++cmake_minimum_required(VERSION 3.0.2)
++project(Lerc)
++
++file(GLOB SOURCES 
++  "src/LercLib/*"
++  "src/LercLib/Lerc1Decode/*"
++)
++
++add_library(LercLib ${SOURCES})
++set_property(TARGET LercLib PROPERTY CXX_STANDARD 11)
```

Comment:
The upstream cmake script requires the c++ 17 standard.
Is upstream too stringent?
Should `self.settings.compiler.cppstd` be checked in `configure()`?

Reply:
C++17 is not required, it's a mistake in their CMakeLists, or maybe they have introduced C++17 features in latest commits.